packages/device_info_plus/device_info_plus/lib/device_info_plus.dart

```diff
@@ -3,12 +3,15 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io';
```

Comment:
Doesn't this throw on web?

Reply:
Not anymore. There is still a problem with some cached compiler info, you cannot mix web and non-web builds (technically speaking runs, not builds) on a single cache. For example if you do:
```console
$ flutter run --device-id macos
```
then
```console
$ flutter run --device-id chrome
```
fails with a bunch of errors referring not only to `dart:io`, but hundreds of errors for `dart:ffi` and `package:win32` which are transient dependencies from `device_info_plus_windows`. This is a result of merging #598. To mitigate it you have to `flutter clean` every time you switch running web vs. non-web platforms. When you do `flutter build web` this is not a problem, because it uses correct compiler every time it's invoked and does not depend on cached stuff.